### PR TITLE
[@types/parse] Add additional supported aggregation options

### PR DESCRIPTION
**Pull request by egmacke**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

### CODE REVIEW

File: types/parse/index.d.ts [hunk 1]
```diff
@@ -832,6 +832,13 @@ declare global {
                 sort?: { [key: string]: 1 | -1 } | undefined;
                 // Sample documentation: https://docs.mongodb.com/v3.2/reference/operator/aggregation/sample/
                 sample?: { size: number } | undefined;
+                count?: string | undefined;
```
Comment (RyanCavanaugh) [inline types/parse/index.d.ts:835]: Count is a string??

**Review by RyanCavanaugh — CHANGES_REQUESTED** [review]
Please add tests

File: types/parse/index.d.ts (hunk 1 above)
Comment (egmacke) [reply types/parse/index.d.ts:836]: Yes - it actually defines the name of the field that is used to store the count value as part of the response. Have updated with a link to the docs

**Review by jakebailey — APPROVED** [review]

**Comment by egmacke** [comment]
Ready to merge

**Pull request merged**